components/admin_console/custom_plugin_settings/custom_plugin_settings.jsx

```diff
@@ -4,13 +4,15 @@
 import SchemaAdminSettings from 'components/admin_console/schema_admin_settings.jsx';
 
 export default class CustomPluginSettings extends SchemaAdminSettings {
-    componentWillReceiveProps = (nextProps) => {
-        if (nextProps.schema !== this.props.schema) {
-            this.setState(this.getStateFromConfig(nextProps.config, nextProps.schema));
-        }
+    constructor(props) {
+        super(props);
+        this.isPlugin = true;
     }
 
-    getConfigFromState = (config) => {
+    // The schema will never change so we want to skip resetting the state from config
+    componentWillReceiveProps = () => {} //eslint-disable-line no-empty-function
```

Comment:
The schema object was never the same and this was causing issues

Reply:
Why leave the empty function?

Reply:
It's to override the default function of the parent class

Reply:
If you refresh the page, it doesn't load the configuration, maybe the problem is here.

Reply:
I try reverting the `componentWillReceiveProps`, and it load correctly the configuration.

Reply:
The configuration loads correctly, but then you can't change any settings (the reason I made this change in the first place). I can update to load the schema only once.